src/features/surveys/pages/PublicSurveyPage.tsx

```diff
@@ -83,7 +85,7 @@ const PublicSurveyPage: FC<PublicSurveyPageProps> = ({ survey, user }) => {
     <Box sx={{ display: 'flex', justifyContent: 'center', width: '100%' }}>
       <Box
         sx={{
-          maxWidth: '37.5rem',
+          maxWidth: isMobile ? '100dvw' : '37.5rem',
```

Comment:
Is `maxWidth` necessary for the `wordBreak` to take effect?

Reply:
I changed from `100dvw` to `undefined` insted, that works just as well on mobile. Or now that I'm writing I'm thinking I'm not sure that's what you meant. Did you mean to remove the `maxWidth` property? To replace with `width`?